src/main/java/net/dries007/tfc/common/blocks/ThatchBlock.java

```diff
@@ -0,0 +1,65 @@
+package net.dries007.tfc.common.blocks;
+
+import net.dries007.tfc.client.ClimateRenderCache;
+import net.dries007.tfc.util.Climate;
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.Blocks;
+import net.minecraft.block.FireBlock;
+import net.minecraft.block.material.Material;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.particles.ParticleTypes;
+import net.minecraft.util.Direction;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.shapes.ISelectionContext;
+import net.minecraft.util.math.shapes.VoxelShape;
+import net.minecraft.util.math.shapes.VoxelShapes;
+import net.minecraft.util.math.vector.Vector3d;
+import net.minecraft.world.IBlockReader;
+import net.minecraft.world.World;
+
+import javax.annotation.Nullable;
+
+public class ThatchBlock extends Block
+{
+    public ThatchBlock()
+    {
+        super(Properties.of(Material.REPLACEABLE_PLANT).strength(0.6F, 0.4F).noOcclusion());
+    }
+
+    @Override
+    @SuppressWarnings("deprecation")
+    public void entityInside(BlockState state, World worldIn, BlockPos pos, Entity entityIn)
+    {
+        entityIn.makeStuckInBlock(state, new Vector3d(0.3D, (double)0.24F, 0.3D));
```

Comment:
```suggestion
        entityIn.makeStuckInBlock(state, new Vector3d(0.3D, 0.24, 0.3D));
```

bad vanilla decompile copypasta